action.yml

```diff
@@ -19,6 +19,9 @@ inputs:
     description: "Ignore model alias matching"
     required: false
     default: false
+  ATLAN_CONFIG:
+    description: "Atlan CLI config file location"
+    required: false
```

Comment:
If it is not required, should we consider of default value?

Reply:
- Since this action also contains dbt impact analysis, making this field required would mean users have to pass `ATLAN_CONFIG` even if they just want to run dbt impact analysis.
- Setting a default value will trigger contract impact analysis, where dbt impact analysis should only run.
- My initial thought was to not run contract impact analysis, unless a user explicitly passes `ATLAN_CONFIG`